gulp/config.js

```diff
@@ -50,6 +50,9 @@ module.exports = function () {
     fonts: {
       src:  basePaths.assets + 'fonts/',
       dest: basePaths.dest + 'assets/fonts/'
+    },
+    revManifest: {
```

Comment:
there is the `manifest.json` (used for example in PWAs). I thinkt this is more clear what it is. Do you agree?